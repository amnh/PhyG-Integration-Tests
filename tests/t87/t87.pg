--build(distance, rdwag, block), inputting tcm and nucleotide files
set(seed:1634561640)
read(nucleotide:"chel-prealigned.fas", tcm:"sg1t4.mat")
read(nucleotide:"chel_cox1aln.fasta", tcm:"sg2t1.mat")
build(distance, rdWag, block, cun, displaytrees:3)
report("t87.csv", data, overwrite)
report("t87.tre", newick, graphs, overwrite)