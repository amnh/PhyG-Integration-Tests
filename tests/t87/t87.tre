(Alentus:153.5,((Thermobius:122.5,(Rhiphicephalus:146.0,((Pauroctonus:98.5,((Amblypygid:134.0,Vonones:106.0)HTU27:39.0,(Limulus:91.5,(Chanbria:109.0,Mastigoproctus:103.0)HTU30:51.0)HTU29:28.5)HTU26:46.5)HTU22:61.5,(Artemia:179.5,(Gea:166.0,Thelichoris:118.0)HTU28:49.0)HTU25:40.5)HTU24:44.5)HTU31:47.5)HTU23:60.5,((Americhernus:221.0,Aportus:117.5)HTU19:93.0,(Hypochilus:99.0,(Centruoides:89.5,Hadrurus:111.0)HTU18:69.5)HTU20:57.0)HTU32:26.0)HTU21:150.5)HTU17[1764.0];
(Alentus:153.5,((Thermobius:122.5,(Rhiphicephalus:146.0,((Pauroctonus:98.5,((Amblypygid:134.0,Vonones:106.0)HTU27:39.0,(Limulus:91.5,(Chanbria:109.0,Mastigoproctus:103.0)HTU30:51.0)HTU29:28.5)HTU26:46.5)HTU22:61.5,(Artemia:179.5,(Gea:166.0,Thelichoris:118.0)HTU28:49.0)HTU25:40.5)HTU24:44.5)HTU31:47.5)HTU23:60.5,((Americhernus:221.0,Aportus:117.5)HTU19:93.0,(Hypochilus:99.0,(Centruoides:89.5,Hadrurus:111.0)HTU18:69.5)HTU20:57.0)HTU32:26.0)HTU21:150.5)HTU17[1764.0];
(Alentus:189.5,((Thermobius:141.5,(Rhiphicephalus:146.5,((Artemia:169.5,(Gea:174.0,Thelichoris:114.0)HTU28:48.5)HTU25:48.0,((Amblypygid:124.5,Vonones:112.5)HTU27:39.0,(Limulus:92.0,(Chanbria:112.5,Mastigoproctus:107.5)HTU30:50.5)HTU29:22.0)HTU26:37.0)HTU24:37.0)HTU31:28.5)HTU23:43.5,((Pauroctonus:110.5,(Americhernus:226.5,Aportus:123.5)HTU20:76.0)HTU18:37.0,(Hypochilus:100.0,(Centruoides:93.0,Hadrurus:107.5)HTU19:65.0)HTU21:42.0)HTU32:14.0)HTU22:157.0)HTU17[1756.0];
