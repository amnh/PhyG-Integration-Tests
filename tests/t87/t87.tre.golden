(Alentus:123.5,(Chanbria:104.0,((Rhiphicephalus:149.0,(Artemia:193.0,((Limulus:126.0,Mastigoproctus:117.0)HTU29:30.0,(Hypochilus:115.0,Thelichoris:91.0)HTU30:41.0)HTU26:35.0)HTU28:21.0)HTU27:43.5,((Gea:162.5,((Amblypygid:82.0,(Americhernus:171.5,Aportus:175.0)HTU20:73.0)HTU19:66.5,(Vonones:103.0,(Pauroctonus:97.0,Thermobius:157.0)HTU23:49.5)HTU24:41.5)HTU18:72.5)HTU21:39.0,(Centruoides:88.0,Hadrurus:143.0)HTU22:76.0)HTU32:49.0)HTU25:45.0)HTU31:181.5)HTU17[1740.0];
(Alentus:104.0,(Chanbria:91.5,((Rhiphicephalus:142.5,(Artemia:165.5,(Mastigoproctus:98.5,(Hypochilus:94.5,Thelichoris:73.5)HTU30:54.5)HTU27:43.5)HTU29:31.0)HTU28:50.5,((Gea:160.5,((Limulus:81.5,(Vonones:96.5,(Pauroctonus:94.5,Thermobius:148.5)HTU24:49.5)HTU25:27.0)HTU19:27.5,(Amblypygid:107.0,(Americhernus:189.0,Aportus:169.0)HTU21:87.0)HTU20:67.5)HTU18:50.0)HTU22:48.0,(Centruoides:75.0,Hadrurus:113.0)HTU23:79.5)HTU32:47.0)HTU26:36.5)HTU31:112.0)HTU17[1734.0];
(Alentus:179.5,(Chanbria:90.0,((Rhiphicephalus:148.5,(Artemia:182.5,((Gea:162.0,(Hypochilus:75.5,Thelichoris:109.5)HTU30:37.0)HTU25:38.5,(Limulus:103.0,Mastigoproctus:116.0)HTU29:34.5)HTU26:41.5)HTU28:24.0)HTU27:45.5,(((Amblypygid:82.0,(Americhernus:233.0,Aportus:124.0)HTU20:51.0)HTU19:37.0,(Vonones:84.0,(Pauroctonus:85.0,Thermobius:145.0)HTU22:55.0)HTU23:28.0)HTU18:48.0,(Centruoides:75.0,Hadrurus:124.0)HTU21:78.0)HTU32:46.5)HTU24:44.5)HTU31:174.0)HTU17[1719.0];
